docs/changelog-fragments.d/957.internal.md

```diff
@@ -0,0 +1,9 @@
+Added new subcommand *settings* to output all current ansible-navigator
+settings to the menu or to stdout.
```

Comment:
Changelog itself is a user-oriented document. And for such texts, the writing style strongly recommends to avoid shortening stuff and using clear full names:
```suggestion
settings to the menu or to standard output.
```

Reply:
Fixed this.